python_modules/dagster/dagster/_core/snap/node.py

```diff
@@ -298,120 +298,114 @@ def get_output_snap(self, name: str) -> OutputDefSnap:
-class SolidDefinitionsSnapshot(
+@whitelist_for_serdes(
+    storage_name="SolidDefinitionsSnapshot",
+    storage_field_names={
+        "op_def_snaps": "solid_def_snaps",
+        "graph_def_snaps": "composite_solid_def_snaps",
+    },
+)
+class NodeDefsSnapshot(
     NamedTuple(
-        "_SolidDefinitionsSnapshot",
+        "_NodeDefsSnapshot",
         [
-            ("solid_def_snaps", Sequence[SolidDefSnap]),
-            ("composite_solid_def_snaps", Sequence[CompositeSolidDefSnap]),
+            ("op_def_snaps", Sequence[OpDefSnap]),
+            ("graph_def_snaps", Sequence[GraphDefSnap]),
         ],
     )
 ):
     def __new__(
         cls,
-        solid_def_snaps: Sequence[SolidDefSnap],
-        composite_solid_def_snaps: Sequence[CompositeSolidDefSnap],
+        op_def_snaps: Sequence[OpDefSnap],
+        graph_def_snaps: Sequence[GraphDefSnap],
     ):
-        return super(SolidDefinitionsSnapshot, cls).__new__(
+        return super(NodeDefsSnapshot, cls).__new__(
             cls,
-            solid_def_snaps=sorted(
-                check.sequence_param(solid_def_snaps, "solid_def_snaps", of_type=SolidDefSnap),
-                key=lambda solid_def: solid_def.name,
+            op_def_snaps=sorted(
+                check.sequence_param(op_def_snaps, "op_def_snaps", of_type=OpDefSnap),
+                key=lambda op_def: op_def.name,
             ),
-            composite_solid_def_snaps=sorted(
+            graph_def_snaps=sorted(
                 check.sequence_param(
-                    composite_solid_def_snaps,
-                    "composite_solid_def_snaps",
-                    of_type=CompositeSolidDefSnap,
+                    graph_def_snaps,
+                    "graph_def_snaps",
+                    of_type=GraphDefSnap,
                 ),
-                key=lambda comp_def: comp_def.name,
+                key=lambda graph_def: graph_def.name,
             ),
         )
 
 
-def build_solid_definitions_snapshot(pipeline_def: PipelineDefinition) -> SolidDefinitionsSnapshot:
+def build_node_defs_snapshot(pipeline_def: PipelineDefinition) -> NodeDefsSnapshot:
     check.inst_param(pipeline_def, "pipeline_def", PipelineDefinition)
-    solid_def_snaps = []
+    op_def_snaps = []
     graph_def_snaps = []
     for node_def in pipeline_def.all_node_defs:
         if isinstance(node_def, OpDefinition):
-            solid_def_snaps.append(build_core_solid_def_snap(node_def))
+            op_def_snaps.append(build_op_def_snap(node_def))
         elif isinstance(node_def, GraphDefinition):
-            graph_def_snaps.append(build_composite_solid_def_snap(node_def))
+            graph_def_snaps.append(build_graph_def_snap(node_def))
         else:
             check.failed(f"Unexpected NodeDefinition type {node_def}")
 
-    return SolidDefinitionsSnapshot(
-        solid_def_snaps=solid_def_snaps,
+    return NodeDefsSnapshot(
+        op_def_snaps=op_def_snaps,
         # update when snapshot renames happen
```

Comment:
you can remove this comment since these names are now aligned - dont think theres any other special meaning
https://dagster.phacility.com/D8943